### PR TITLE
feat: changed the API to the oficial strapi with the right data, also…

### PR DESCRIPTION
**Pull request by J-HernandezM**
BREAKING CHANGE: the new structure of the response differs with the previous one

vercel checks wont work for the moment since we need first to recieve all the missing attributes from the API (my boss is on a birthday so ill have to wait 'til tomorrow to do so)

### CODE REVIEW

File: src/context/products.jsx [hunk 1]
```diff
@@ -2,20 +2,33 @@
 
 import React from "react";
 import { useFetch } from "../shared/useFetch";
+import { sanitizeApiResponse } from "../shared/sanitizeApiResponse";
 
 export const ProductsContext = React.createContext();
 
+const myHeaders = new Headers();
+myHeaders.append("Authorization", "Bearer 932ae15adebc593dd0243c41f99474e536809316b75d9fa2d38c20790db1d0f0f6c8d005ed923248ee04fa89d476d88208c18706cd6439d83382bc69e209e5f7060bbe16fdd071b9bddedd8eff8e207fba37655a70f95652760b2328e09caf080eb3975f391eaa8a71c1af8d1edf14463d40ecd93c00d48c15572def7b05189c");
```
Comment (santb08) [inline src/context/products.jsx]: get the token from the environment

use `.env` file (if you don't have one, create one)

don't expose this API token on GitHub, now I'll have to create you another one

File: src/context/products.jsx [hunk 2]
```diff
@@ -2,20 +2,33 @@
...
+};
+
 export default function ProductsProvider({children}){
     const {
         data, 
         loading, 
         error, 
         handleCancelRequest
-    } = useFetch('https://api.escuelajs.co/api/v1/products')
+    } = useFetch('http://146.190.145.102:1337/api/products', requestOptions)
```
Comment (santb08) [inline src/context/products.jsx]: You can also have an `API_URL` environment variable in your `.env` file

> https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables

**Review by santb08 — CHANGES_REQUESTED** [review]

File: src/context/products.jsx (hunk 1 above)
Comment (J-HernandezM) [reply src/context/products.jsx]: im sorry, my bad

**Comment by santb08** [comment]
`shared/sanitizeApiResponse.js` coverage
image

**Pull request merged**